Models/NHES/Fluid/Vessels/package.mo

```diff
@@ -1,9 +0,0 @@
-within NHES.Fluid;
```

Comment:
Can you check to make sure this should be deleted?

Reply:
It should not be deleted. I have added a heat exchanger I would like to keep and an effectiveness function for heat exchangers.